pkg/sdk/accounts_integration_test.go

```diff
@@ -63,6 +60,13 @@ func TestInt_AccountCreate(t *testing.T) {
 		}
 		err = client.Accounts.Create(ctx, accountID, opts)
 		require.NoError(t, err)
+
+		t.Logf("sleeping for 5 seconds to wait for account to be created")
+		for i := 0; i < 5; i++ {
+			t.Logf("%d", 5-i)
+			time.Sleep(1 * time.Second)
+		}
```

Comment:
maybe you could do 3 tries of ShowByID instead of sleeping (Something similar to Retries you've done in this change) ? I would prefer it that way, because sometimes it could proceed with the test faster than 5 sec and won't add additional seconds to the integration tests

Reply:
done